.ci/install_runtime.sh

```diff
@@ -87,13 +87,10 @@ if [ "$KATA_HYPERVISOR" == "qemu" ]; then
 elif [ "$KATA_HYPERVISOR" == "nemu" ]; then
 	echo "Configure Nemu as Kata Hypervisor"
 	sudo sed -i -e 's|machine_type = "pc"|machine_type = "virt"|' "${runtime_config_path}"
-	sudo sed -i -e 's|firmware = ""|firmware = "/usr/share/nemu/firmware/OVMF.fd"|' "${runtime_config_path}"
+	sudo sed -i -e 's|firmware = ""|firmware = "/usr/local/share/kata-nemu/OVMF.fd"|' "${runtime_config_path}"
 	case "$arch" in
 	x86_64)
-		sudo sed -i -e "s|\"/usr/bin/qemu-lite-system-${arch}\"|\"/usr/local/bin/qemu-system-${arch}_virt\"|" "${runtime_config_path}"
-		;;
-	aarch64)
-		sudo sed -i -e "s|\"/usr/bin/qemu-lite-system-${arch}\"|\"/usr/local/bin/qemu-system-${arch}\"|" "${runtime_config_path}"
+		sudo sed -i -e "s|\"/usr/bin/qemu-lite-system-${arch}\"|\"/usr/local/bin/nemu-system-${arch}\"|" "${runtime_config_path}"
```

Comment:
Nit: Again, I appreciate you're keeping this change similar to what was there before, but I think we should be trying to use `crudini` for all config file changes now tbh as the syntax is much clearer and the tool is already being installed for all distros.